README.md

```diff
@@ -47,8 +47,13 @@ An example event can be created using the helper script, `make produce`.
 | HEALTHCHECK_INTERVAL         | 30s                               | Time between self-healthchecks (`time.Duration` format)
 | HEALTHCHECK_CRITICAL_TIMEOUT | 90s                               | Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format)
 | KAFKA_ADDR                   | "localhost:9092"                  | The address of Kafka (accepts list)
-| KAFKA_OFFSET_OLDEST          | true                              | Start processing Kafka messages in order from the oldest in the queue
-| KAFKA_NUM_WORKERS            | 1                                 | The maximum number of parallel kafka consumers
+| KAFKA_VERSION                | `1.0.2`                  | The version of Kafka
+| KAFKA_CONSUMER_WORKERS       | 1                        | The maximum number of parallel kafka consumers
```

Comment:
where is this in config.go (I think you renamed it, from `KAFKA_NUM_WORKERS` for some reason)?